src/test/resources/debug-config/logback.xml

```diff
@@ -1,27 +1,18 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
 <configuration>
 	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-		<!-- encoders are assigned the type
-         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
 		<encoder>
-			<pattern>[%date{ISO8601}] %-5level %msg %n</pattern>
```

Comment:
same here. and why is there a need to change the log pattern? these log files are used in the manual test phase and are (IMHO) fairly important. Also the changes below of turning off the logging at all seems completely weird. Either undo this change or give a very good explanation for doing this!